api/api/roles/serializers.py

```diff
@@ -8,3 +8,10 @@ class Meta:
         model = Role
         fields = ['name', 'club', 'user', 'assigned_at', 'active']
         extra_kwargs = {'assigned_at': {'read_only': True}}
+
+
+class ListUserRolesSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Role
+        fields = ['name', 'club', 'assigned_at', 'active']
```

Comment:
Currently `club` would be an integer representing `id` of a club. Do we need any more information about the club in the response?

Reply:
@Aksh-Bansal-dev name would be fine?

Reply:
Yeah, we only need the clubs' names.